seatingchart.py

```diff
@@ -166,19 +176,8 @@ def working_dir_path(name, slug, extension):
 for seat, uni in assignments.iteritems():
     output.writerow(list(uni)[:2] + [seat])
 
-
-#Load picture files
-soup = BeautifulSoup(open(ROSTER_PAGE), "html.parser")
-photos = {}
-for img in soup.findAll(class_="rosterImage"):
-    url = img["src"]
-    uni = img.findParent().findNextSibling().findNextSibling().string
-    photos[uni] = url
```

Comment:
was this the only use of beautiful soup? we can get rid of the import so people don't need to install that library anymore